docs/icicle/supporting-additional-curves.md

```diff
@@ -45,21 +44,19 @@ In order to support a new curves in the binding libraries you first must support
 
 Create a new folder named `icicle-<curve_name>` under the [rust wrappers folder](https://github.com/ingonyama-zk/icicle/tree/main/wrappers/rust/icicle-curves). Your new directory should look like this.
 
-
-* icicle-<curve_name>/
-  * src/
-    * ntt/
-        * mod.rs
-    * msm/
-        * mod.rs
-    * curve.rs
-    * lib.rs
-  * Cargo.toml
-  * build.rs
+- icicle-<curve_name>/
+  - src/
+    - ntt/
+      - mod.rs
+    - msm/
+      - mod.rs
+    - curve.rs
+    - lib.rs
+  - Cargo.toml
+  - build.rs
```

Comment:
Can we use a tree view of this instead of bullet points?
Something like https://github.com/ingonyama-zk/developer-docs/blob/main/docs/icicle/introduction.md?plain=1#L179-L182